zh/_contributing.md

```diff
@@ -1,10 +1,20 @@
 
 # 贡献
 
-Sass指南是我利用空暇时间维持的一个免费项目。不用多说,相信你也能够理解，持续更新、编写文档和其他相关事宜是一件工作量不小的事情。幸运的是，我从许多非常棒的贡献者那里得到了帮助，从而让该规范翻译成了<button type="button" data-modal-show="options-panel" class="link-like">多种语言</button>。请向他们致敬！
+Sass指南是我利用空暇时间维持的一个免费项目。不用多说,相信你也能够理解，持续更新、编写文档和其他相关事宜是一件工作量不小的事情。幸运的是，我从许多非常棒的贡献者那里得到了帮助，从而让该规范翻译成了<label for="aside-toggle" class="link-like">多种语言</label>。请向他们致敬！
 
 现在，如果你想要为它做些贡献，一个很好的建议是：使用社交网络分享它，也可以在 [GitHub repository](https://github.com/HugoGiraudel/sass-guidelines) 上提交修正错误的 issue 或者 pull-request。
 
 在开始之前最后但不是最终的一句话：如果你喜欢这份样式指南，如果它对你或你的团队大有裨益，请考虑支持它，从而鼓励我持续迭代不断改进本文！
 
-{% include partials/contributing.html %}
+<div class="button-wrapper">
```

Comment:
It should be `{% include partials/contributing.html %}` here but I can also fix it myself.